src/exportAuthRecord.ts

```diff
@@ -122,6 +122,36 @@ async function persistAuthRecord(authRecord: Record<string, unknown>): Promise<v
     const authDir = path.join(baseAzureDir, 'ms-azuretools.vscode-azureresourcegroups');
     const authRecordPath = path.join(authDir, 'authRecord.json');
     const gitignorePath = path.join(authDir, '.gitignore');
+    // Write README.md explaining the purpose and contents of authRecord.json
+    const readmePath = path.join(authDir, 'README.md');
+    const readmeContent = [
+        'The `authRecord.json` file is created after authenticating to an Azure subscription from VS Code. For example, via the **Azure: Sign In** command in Command Palette. The directory in which the file resides matches the unique identifier of the [Azure Resources extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureresourcegroups) responsible for writing the file.',
```

Comment:
The first time we use "VS Code" we should expand the name
```suggestion
        'The `authRecord.json` file is created after authenticating to an Azure subscription from Visual Studio Code. For example, via the **Azure: Sign In** command in Command Palette. The directory in which the file resides matches the unique identifier of the [Azure Resources extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureresourcegroups) responsible for writing the file.',
```